packages/csharp/PortingAssistant/PortingAssistant.Api/Application.cs

```diff
@@ -111,18 +118,54 @@ public void SetupConnection(bool console = false)
                 try
                 {
                     var file1 = httpService.DownloadS3FileAsync("newtonsoft.json.json.gz");
```

Comment:
move the hardcoded strings to constants file

Reply:
Hi Sai, it is from original code. I will let original author know this. Resolve it for now. Thanks!